python/setup.py.in

```diff
@@ -66,31 +79,21 @@ if not os.path.isfile(os.path.join('smt_switch', ext_filename)):
         os.environ.setdefault("_PYTHON_HOST_PLATFORM", host_platform)
         arch_flags = "-arch " + arch
         os.environ.setdefault("ARCHFLAGS", arch_flags)
-
     if sys.platform == "win32":
         extra_compile_args += ["-DMS_WIN64"]
```

Comment:
As a side effect, also fixed this being an undefined variable when compiling on Windows.